css/index.css

```diff
@@ -143,4 +250,58 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+
+/*About Page:*/
+
+.Leverage{border-bottom: 1px dashed black;
+}
+.Row1{display:flex;
```

Comment:
If the styles are identical, you can reuse class names instead of typing the same style out multiple times. (Row1, Row2, Column1, Column2)